Makefile

```diff
@@ -10,6 +10,7 @@ endef
 build:
 	cp config/database.yml.example config/database.yml
 	cp config/config.yml.example config/config.yml
+	cp config/initializers/recaptcha.rb.example config/initializers/recaptcha.rb
```

Comment:
This includes recaptcha to our Jenkins pipeline, I don't think this step is needed, unless I'm missing something. @icarito is the one that has control over our Jenkins CI/CD.

Reply:
Thanks for your review @alaxalves - indeed I think this isn't needed in fact it would override the recaptcha variables in staging.